GyroData/Scenes/MotionListScene/MotionListViewController.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  MotionListViewController.swift
+//  GyroData
+//
+//  Created by 이호영 on 2022/12/27.
+//
+
+import UIKit
+
+class MotionListViewController: UIViewController {
+
+    private var motionListTableView = UITableView()
```

Comment:
```suggestion
    private let motionListTableView: UITableView = {
        let tableView = UITableView() 
        tableView.delegate = self
        tableView.dataSource = self
        tableView.register(
            MotionListTableViewCell.self,
            forCellReuseIdentifier: MotionListTableViewCell.cellIdentifier
        )
        tableView.translatesAutoresizingMaskIntoConstraints = false
        return tableView
    }
```

저는 이렇게 설정도 한 번에 해서 만드는 편인데 이건 취향차이라 그대로 해도 괜찮습니다!

Reply:
오! 이렇게 수정해보겠습니다~ ☺️